{"name": "无条件查询", "status": "failed", "statusDetails": {"message": "AssertionError: 响应json节点检查失败，实际结果【0】，预期结果【1】，json节点路径【error_code】", "trace": "get = <function get.<locals>.__get at 0x000001951DF20158>\nset = <function set.<locals>.__set at 0x000001951DF201E0>\n\n    @feature('查询活动接口')\n    @title('无条件查询')\n    @order(order=2)\n    def test_geteventlist01(get,set):\n        '''\n        查询所有活动\n        :return:\n        '''\n        client = Client(url=\"get_eventlist/\",method=METHOD.GET,body_type=BODY_TYPE.FORM,params={\"rstr\":get('rstr')})\n        client.set_headers({\"uid\":get('uid'),\"key\":get('key')})\n        client.send()\n        client.check_status_code(200)\n        client.check_res_times_less_than(1000)\n>       client.check_json_value('error_code',1)\n\ncases\\test_geteventlist.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <core.client.Client object at 0x000001951DE864E0>, path = 'error_code'\nexpected = 1\n\n    @allure.step('检查json节点数据')\n    def check_json_value(self,path,expected):\n        actual_value = self.get_json_value(path)\n        assert str(actual_value) == str(expected), '响应json节点检查失败，实际结果【{actual}】，预期结果【{expected}】，json节点路径【{path}】'\\\n>           .format(actual=actual_value,expected=expected,path=path)\nE       AssertionError: 响应json节点检查失败，实际结果【0】，预期结果【1】，json节点路径【error_code】\n\ncore\\client.py:162: AssertionError"}, "description": "\n    查询所有活动\n    :return:\n    ", "steps": [{"name": "请求接口详细信息", "status": "passed", "attachments": [{"name": "请求地址", "source": "a082b5ca-9c3a-4811-b1db-59c73ecaa3c7-attachment.txt", "type": "text/plain"}, {"name": "请求地址", "source": "9c80c635-9e47-4eb5-945b-3d95c3b84649-attachment.txt", "type": "text/plain"}, {"name": "请求头", "source": "fc182a05-df04-4e88-bc7e-c73a92761e80-attachment.txt", "type": "text/plain"}, {"name": "请求正文", "source": "cb2b3afa-4cad-4d0b-9427-e227c1fafe23-attachment.txt", "type": "text/plain"}, {"name": "响应内容", "source": "8dc481ab-5718-4b6a-9fef-b1946d583c4f-attachment.txt", "type": "text/plain"}], "start": 1595412388718, "stop": 1595412388851}, {"name": "检查响应状态码", "status": "passed", "parameters": [{"name": "status_code", "value": "200"}], "start": 1595412388851, "stop": 1595412388851}, {"name": "检查响应时间", "status": "passed", "parameters": [{"name": "times", "value": "1000"}], "start": 1595412388851, "stop": 1595412388851}, {"name": "检查json节点数据", "status": "failed", "statusDetails": {"message": "AssertionError: 响应json节点检查失败，实际结果【0】，预期结果【1】，json节点路径【error_code】\n", "trace": "  File \"C:\\Users\\chenmeihuan\\AppData\\Local\\Programs\\Python\\Python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"E:\\cmh\\personal\\pycharmworkspace\\interface_fill\\core\\client.py\", line 162, in check_json_value\n    .format(actual=actual_value,expected=expected,path=path)\n"}, "parameters": [{"name": "path", "value": "'error_code'"}, {"name": "expected", "value": "1"}], "start": 1595412388851, "stop": 1595412388851}], "start": 1595412388718, "stop": 1595412388852, "uuid": "7f951a84-47bd-4147-9bdd-dfc9ff456e54", "historyId": "d08f7e1daf8bae270356a158d0d66807", "testCaseId": "058be487299ddbe0eeb3730afebbedb9", "fullName": "cases.test_geteventlist#test_geteventlist01", "labels": [{"name": "feature", "value": "查询活动接口"}, {"name": "tag", "value": "run(order=2)"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_geteventlist"}, {"name": "host", "value": "chenmeihuan-l1"}, {"name": "thread", "value": "4508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_geteventlist"}]}